tests/TestCase/Cache/Engine/MemcachedEngineTest.php

```diff
@@ -446,6 +446,18 @@ public function testConnectIpv6()
     }
 
     /**
+     * test domain starts with u
+     *
+     * @return void
+     */
+    public function testParseServerStringWithU()
+    {
+        $Memcached = new TestMemcachedEngine();
+        $result = $Memcached->parseServerString('udomain.net:13211');
+        $this->assertEquals(array('udomain.net', '13211'), $result);
```

Comment:
Our codesniffer needs a sniff to shout warnings for not using short array syntax :)